src/presets/static.ts

```diff
@@ -8,6 +8,7 @@ export const _static = defineNitroPreset({
   },
   prerender: {
     crawlLinks: true,
+    routes: ["/404.html"],
```

Comment:
This only makes sense for github i guess for default behavior (not sure what other static providers respect `404.html` as fallback)